src/bridge/IrcBridge.ts

```diff
@@ -523,6 +523,7 @@ export class IrcBridge {
             logMessage(req, "FAILED");
             stats.request(isFromIrc, "fail", req.getDuration());
             this.logMetric(req, "fail");
+            BridgeRequest.HandleExceptionForSentry(req, "failed");
```

Comment:
this should be `fail`, to match the others? FAILED, fail, fail, failed

Reply:
Hmm, tricky. From an english standpoint, I liked state="failed" more than state="fail". However, it does introduce some inconsistency.

Reply:
I don't mind which, just think they should be consistent